json_parser.rb

```diff
@@ -1,13 +1,14 @@
 require "json"
 
 class Parser
-  attr_reader :json_body
+  # attr_reader :json
+  #
+  # def initialize(json)
+  #   @json = json
+  # end
 
-  def initialize(json_body)
-    @json_body = json_body
-  end
-
-  def to_hash
+  def to_hash(json)
+    json_body = json.body
```

Comment:
is this variable necessary? json_body and json.body are effectively the same readbility wise
Let's take it out and just parse the body directly